tests/eas/fork_migration.config

```diff
@@ -0,0 +1,52 @@
+{
+    "modules"  : [ "bl" ],
+    "tools"    : [ "rt-app" ],
+    "ftrace" : { // TODO make these inherited
+        "events" : [
+            "sched_energy_diff",
+            "sched_load_avg_task",
+            "sched_load_avg_cpu",
+            "sched_migrate_task",
+            "sched_switch"
+        ],
+    },
+    "wloads" : {
+        // Create N 100% tasks and M 10% tasks which run in parallel, where N is
+        // the number of big CPUs and M is the number of LITTLE CPUs.
+        "fmig" : {
+            "type" : "rt-app",
+            "conf" : {
+                "class" : "profile",
+                "params" : {
+                    "small" : {
+                        "kind" : "Periodic",
+                        "params" : {
+                            "duty_cycle_pct": 10,
+                            "duration_s": 5,
+                            "period_ms": 10,
+                        },
+                        "tasks" : "little", // Create one task for each LITTLE CPU
```

Comment:
That's just a note for the further clean-up of the hardcoded big.LITTLE values... here we should probably provide the possibility to define a string which is used to match the CPU names defined in platform.json.
For example on a X20 platform, which has:
```python
	"board" : {
		"cores" : [
			"A53_0", "A53_0", "A53_0", "A53_0",
			"A53_1", "A53_1", "A53_1", "A53_1",
			"A72_0", "A72_0",
		],
	},
```
we could use:
- "A53_0" to generate as many tasks as the cores of the first cluster
- "A53" to generate as many tasks as the cores in the first two clusters

But that's just a note for another rework ;-)